src/v2/Apps/Home/Components/HomeTroveArtworksRail.tsx

```diff
@@ -33,6 +41,16 @@ export const HomeTroveArtworksRail: React.FC<HomeTroveArtworksRailProps> = ({
       subTitle="A weekly curated selection of the best works on Artsy by emerging and sought after artists."
       viewAllLabel="View All Works"
       viewAllHref="/gene/trove"
+      viewAllOnClick={() => {
+        const trackingEvent: ClickedArtworkGroup = {
+          action: ActionType.clickedArtworkGroup,
+          context_module: ContextModule.troveArtworksRail,
+          context_page_owner_type: OwnerType.home,
+          destination_page_owner_type: OwnerType.gene,
```

Comment:
We could technically add a destination_page_owner_slug and destination_page_owner_id for the gene here right?

Reply:
ok gotcha!